src/GovUk.Education.ExploreEducationStatistics.Content.Api/Requests/CreateFeedbackRequest.cs

```diff
@@ -0,0 +1,19 @@
+#nullable enable
+using GovUk.Education.ExploreEducationStatistics.Common;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Api.Requests;
+
+public record CreateFeedbackRequest
+{
+    public required string Url { get; set; }
+
+    public string? UserAgent { get; set; }
+
+    public required FeedbackResponseOptions Response { get; set; }
+
+    public string? Context { get; set; }
+
+    public string? Issue { get; set; }
+
+    public string? Intent { get; set; }
```

Comment:
Looks like we're missing server-side validation - we should add FluentValidation rules that should at least match the database constraints.